test/unit/io_managers_test.py

```diff
@@ -234,7 +234,12 @@ def test_handling_view_with_metadata(pudl_sqlite_io_manager_fixture):
 
     # read the view data as a dataframe
     input_context = build_input_context(asset_key=AssetKey(asset_key))
-    pudl_sqlite_io_manager_fixture.load_input(input_context)
```

Comment:
🆘  I need some help here - I'm not a SQLAlchemy wizard 🧙 so I'm having a hard time understanding these mocked/fixtured SQLAlchemy objects

Reply:
I think this is probably a @bendnorman question.

Reply:
`load_input()` is calling `pd.read_sql_table(table_name, con)`, which I think pre-pandas 2.0 worked for reading views but is now failing. We could do `pd.read_sql_query(f"select * from {table_name}", con)` to handle tables and views but I think `read_sql_table` does a better job inferring types. However, we might not run into datatype conversion issues given we are correcting dtypes using `Resource.enforce_schema()` in `load_input()`.

All that to say, I think we should xfail it for now. We aren't actually creating any views right now and we should probably do more testing to make sure the types are conserved.